sql/hive/src/test/scala/org/apache/spark/sql/hive/ShowCreateTableSuite.scala

```diff
@@ -28,11 +28,11 @@ class ShowCreateTableSuite extends QueryTest with SQLTestUtils with TestHiveSing
   import testImplicits._
 
   test("data source table with user specified schema") {
-    withTable("ddl_test1") {
+    withTable("ddl_test") {
```

Comment:
I firstly noticed this bug while writing tests in this test suite. I found that test cases always fail if I use the same table names in multiple test cases. That's why I made the changes to this file as additional tests.